apps/files/models.py

```diff
@@ -48,7 +48,7 @@ def duplicate(self):
             schema=self.schema,
             team=self.team,
         )
-        if self.file:
+        if self.file and self.file.storage.exists(self.file.name):
```

Comment:
Is / was there a case that necessitates this extra check, or is this precautionary?

Reply:
There shouldn't be but it's possible the file get's removed. I did have this happen locally but that's because I cleared the files.